### DISCUSSION TITLE
Consider adding easing for `dragscroll`

### DISCUSSION THREAD

**Discussion by vayqerlukashakkarainen** [vnphanquang/svelte-put#215] [Ideas]
I feel like an easing option for `dragscroll` would be a nice addition since many other libraries (not made for svelte) has it. Basically fading out the delta over time when dragging really hard.

I could contribute on improving `dragscroll` to deal with such a feature.

**Comment by vnphanquang** [comment]
@vayqerlukashakkarainen thank you for the suggestion. That is one nice feature indeed. I had thought about it before the last release, but never got around to actually implement it.

Momentum-based scrolling takes a bit of extra work. I love to do some more research to find the most fit solution for `dragscroll` but i'm quite occupied at the moment. Any further discussion or PR is welcomed. Let me know.

**Reply by vayqerlukashakkarainen** [reply]
Alright, I ping you if I'll get to it. Thank you for your response.
